metric_learn/covariance.py

```diff
@@ -35,11 +35,11 @@ def fit(self, X, y=None):
     y : unused
     """
     X = self._prepare_inputs(X, ensure_min_samples=2)
-    M = np.cov(X, rowvar = False)
-    if M.ndim == 0:
+    M = np.atleast2d(np.cov(X, rowvar = False))
```

Comment:
I added an atleast2d since we don't want the metric to be scalar

Reply:
While we're here, remove the spaces in the `rowvar` keyword argument.

Reply:
I agree, done